client/App.jsx

```diff
@@ -32,12 +33,16 @@ React.Component {
         })
     }
 
+    onClick(event) {
+        const target = event.target;
+    }
+
     render() {
         this.get()
         return (
             <div id="Marks-main-container">
                 <div id="Marks-side-images">
-                <SideImages />
+                <SideImages imgId={this.state.selectedItem} onClick={this.onClick.bind(this)} numOfImgs={this.state.numOfImgs} imageNumber={this.state.imageNumber} />
```

Comment:
I think you are doing .bind on this function twice, once in the constructor and once here...